src/Common/ProcessExecutor.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Robo\Common;
+
+use Robo\Contract\TaskInterface;
+use Robo\Robo;
+use Robo\Task\BaseTask;
+
+class ProcessExecutor extends BaseTask implements TaskInterface
```

Comment:
It doesn't seem right for ProcessExecutor to extend BaseTask or implement TaskInterface; since this class is not intended to be used as a Task, it should be satisfy `isa` for either BaseTask or TaskInterface.

Since ProcessExeuctor is not used as a task, it shouldn't need any of the `inflect` methods / interfaces. It would therefore be a lot better if ProcessExecutor simply `use`d the traits from BaseTask that it needs, and forgo extending this class.